### ISSUE TITLE
Setup OpenSearch

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#11]
## User story
1. As a user
2. I want an OpenSearch node
3. So that i can use it to query searches for a large pile of data

## Acceptance criteria
* Make sure we have a running OpenSearch node inside our docker container #8 
* Make all initial configurations that may be necessary

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
